Octokit.Tests/Reactive/ObservableIssuesClientTests.cs

```diff
@@ -337,6 +337,56 @@ public void EnsuresArgumentsNotNull()
         }
     }
 
+    public class TheLockIssueMethod
+    {
+        [Fact]
+        public void LockIssue()
+        {
+            var gitHubClient = Substitute.For<IGitHubClient>();
+            var client = new ObservableIssuesClient(gitHubClient);
+
+            client.LockIssue("fake", "repo", 42);
+            gitHubClient.Issue.Received().LockIssue("fake", "repo", 42);
+        }
+
+        [Fact]
+        public void EnsuresArgumentsNotNull()
+        {
+            var gitHubClient = Substitute.For<IGitHubClient>();
+            var client = new ObservableIssuesClient(gitHubClient);
+
+            Assert.Throws<ArgumentNullException>(() => client.LockIssue(null, "name", 42));
+            Assert.Throws<ArgumentException>(() => client.LockIssue("", "name", 42));
+            Assert.Throws<ArgumentNullException>(() => client.LockIssue("owner", null, 42));
+            Assert.Throws<ArgumentException>(() => client.LockIssue("owner", "", 42));
+        }
+    }
+
+    public class TheUnlockIssueMethod
+    {
+        [Fact]
+        public void UnlockIssue()
```

Comment:
Wording for the unit tests should read TheBlahMethod....DoesSomething

So...
TheUnlockMethod->Unlock**s**Issue

(note Unlocks rather than Unlock)